.travis.yml

```diff
@@ -1,8 +1,15 @@
 language: python
-python:
-  # We don't actually use the Travis Python, but this keeps it organized.
-  - "3.5"
-  - "3.6"
+sudo: false
+env:
+   global:
+     - secure: "t399QOE83FONpr3PslOu6E28TwJUr6Kywt8JYVae22JCp2l703KYru4Bw3VFOfJSv2A1BISrU5Q6z5xu8BU3qU9V9UtPX7r6iVnK/Jqhxh+jjLA2tglBDUfWLYABC27UgaqDyGs2b9K+q/5c1Fv7jUPiZ+a53ly7Zwg7pu679KvFhbhBhPooSGxkyY+OISNmNr2VF1E34BxzWEaqD0yqjX8DsJEvUjPRwhPxTCQ3h4+BahhXWVZD0+I64/ATvf/4dsjK5LJ9nQLWrgaK9zEklNOw4XHJDthCQ22s2Vx6VlCDmUeHQrGGtqOK2n0atPWr6tOGkRtu2r9RArUt92tcYWt8cUIBFGP18UBRvRBdYkhvqysIUgLA4TmbI/H0uMtHnmvJ3AVjb6oRLlfQWUwneMm+LcawdvVGmVh7Jbo3c7T6KUaL2YWkyUuoao2IshAhbB09OG5QYcIFAsB68COPhVoP9ck88+Z7QjpBGox9epJnBUYedTAyijX7e23XTOgTeSXKxv3tnYdVq8gLyjhcp2E53W3PplRjawAaCuyAgyFR14tniCV0Pc6XehPa2ANCLbuqWLCQZac5KyMQ4ENQMzWoIWD0+wGnaGdHLBDPcT0TZogE9uvdTbLMnhhQVRebXu3gbCDhWAzWTltU2QIyYBsCVGQMXmxDmo3zGHXJnBg="
+
+matrix:
+   include: 
+      - python: 3.5
+      - python: 3.6
+      - python: 3.6
```

Comment:
Why twice?

Reply:
This is how the `bluesky` one is setup. Beyond that I have to admit I was just following suit. I guess it is easy to look at the build-log and see if 3.6 passed and 3.6 BUILD_DOCS failed there is something wrong `doctr`?